SPIKE.md

```diff
@@ -0,0 +1,126 @@
+# Run LB3 tests from LB4 when LB3 is mounted on the LB4 app
+
+Since LoopBack 4 offers a way to mount LoopBack 3 applications on a LoopBack 4
+project with the use of
+[`@loopback/booter-lb3app`](https://github.com/strongloop/loopback-next/tree/master/packages/booter-lb3app),
+there should also be a way for users to run their tests as part of LoopBack 4's
```

Comment:
Let's please clarify that `their tests` are referring to the LB3 application tests.